synapse/common.py

```diff
@@ -668,7 +694,7 @@ def setlogging(mlogger, defval=None, structlog=None):
             logging.basicConfig(level=log_level, handlers=(handler,))
         else:
             logging.basicConfig(level=log_level, format=s_const.LOG_FORMAT)
-        mlogger.info('log level set to %s', log_level)
+        mlogger.info('log level set to %s', s_const.LOG_LEVEL_INVERSE_CHOICES.get(log_level))
```

Comment:
May need to default to the value here to avoid printing None on a custom int log level

Reply:
The normLogLevel won't return integers which have a unknown value.